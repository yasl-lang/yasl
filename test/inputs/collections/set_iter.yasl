
const s = collections.set(.a, .a2, false, true, 1, 2, 3, .a, .bbbbb, .dsdasdasd, 4)

for e in s {
    echo e
}
