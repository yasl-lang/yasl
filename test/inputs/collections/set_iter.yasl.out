false
2
a
1
true
4
bbbbb
dsdasdasd
3
a2
